File: OOP/3_private_encaptulaiton.cpp
#include <iostream>
#include <string>
using namespace std;

class Account{
    private:
        int account_number;
        double balance;
    public:
        Account(int account_number,int balance) : account_number(account_number),balance(balance) {} ;
    
        string get_account_number(){
            return "Your Account Number is: " + to_string(account_number);
        }

        string get_balance(){
            return "Your Current Balance is: " + to_string(balance);
        }
};

int main(){
    Account obj1(12543675,30000);
    
    cout<< obj1.get_account_number() << endl;
    cout<< obj1.get_balance() << endl;

    return 0;
}

File: OOP/3_private_encaptulaiton.py
class Account:
    def __init__(self,account_number,balance):
        self.__account_number = account_number
        self.__balance = balance

    def get_account_number(self):
        return f"Your account number is: {self.__account_number}"
    
    def get_balance(self):
        return f"Your current balance is: Tk. {self.__balance}"
    

# here account number and balance is example of private encapsulation
obj1 = Account(12543675,30000)
print(obj1.get_account_number())
print(obj1.get_balance())

